qiita_pet/support_files/doc/source/tutorials/portals.rst

```diff
@@ -0,0 +1,38 @@
+.. _portals:
+
+.. index:: portals
+
+User Portals
+============
+
+User portals allow a user to cordon off specific studies and analyses into their own user interface, which we call a portal, selectively slicing the data available through that portal.
+
+The main `QIITA` portal gives access to all studies in the database, while adding studies to a specific portal and changing the config file to point to the portal will show the same interface, but with only the selected studies. To change the portal being accessed, simply change the `portal` setting in the qiita config file. This must correspond to a portal name in the database.
```

Comment:
Would you mind writing this document to fit within 80 columns? We don't have a guideline on this so, it shouldn't be a blocker but I definitely think it will make reviewing changes to this file easier (and the rest of the documents are formatted like that).

Can you replace all the occurrences of \`QIITA\` for just Qiita.